tools/js-optimizer.js

```diff
@@ -8355,6 +8367,16 @@ function applyImportAndExportNameChanges(ast) {
         }
         return item;
       });
+    } else if (type === 'var') {
+      var children = node[1];
+      var target = children[0][0];
+      var value = children[0][1];
+      if (value && isAsmUse(value)) {
+        var name = value[2][1];
+        if (mapping[name]) {
+          value[2][1] = mapping[name];
+        }
+      }
```

Comment:
In Wasm exports come in form of

```js
var _free, _llvm_minnum_f32, _main, _malloc, _memcpy, _sbrk, dynCall_idi;
WebAssembly.instantiate(Module["wasm"], imports).then((function(output) {
 var asm = output.instance.exports;
 _free = asm["N"];
 _llvm_minnum_f32 = asm["O"];
 _main = asm["P"];
 _malloc = asm["Q"];
 _memcpy = asm["R"];
 _sbrk = asm["S"];
 dynCall_idi = asm["T"];
 for (var i in __ATINIT__) __ATINIT__[i].func();
 ready();
}));
```

and in asm.js they look like

```js
var asm =Module["asm"](asmGlobalArg, Module.asmLibraryArg, buffer);
var _free = asm["_free"];
var _main = asm["_main"];
var _malloc = asm["_malloc"];
```

hence the changes to JS optimizer to recognize the different syntaxes.